docs/plugin-writer/tasks/add-remove.rst

```diff
@@ -1,8 +1,27 @@
 Adding and Removing Content
 ===========================
 
-TODO: RepositoryVersion
-https://pulp.plan.io/issues/3220
+Repositories have versions. A new respository version is created when its set of content units
```

Comment:
maybe: /have versions/have immutable versions/

Not sure if that's too technical?

Reply:
I think "A new immutable respository version is created when its set of content units changes" is sufficient. At the start of the guide, I will point to the overview/concepts of the code docs as prerequisite knowledge.

Reply:
Should I clarify what immutable means in this context (can't be changed, but can be deleted)? And add some details about how deleting versions would squash up the changes into the next version..

Reply:
@asmacdo cool, didn't see your comment. I will do that instead